src/pages/RepoPage/FailedTestsTab/FailedTestsPage/MetricsSection/MetricsSection.tsx

```diff
@@ -6,6 +6,25 @@ import Icon from 'ui/Icon'
 import { MetricCard } from 'ui/MetricCard'
 import { Tooltip } from 'ui/Tooltip'
 
+import { useTestResultsAggregates } from '../hooks/useTestResultsAggregates'
+
+const PercentBadge = ({ value }: { value: number }) => {
+  let variant: 'success' | 'danger' = 'success'
+  let prefix = ''
+
+  if (value > 0) {
+    variant = 'danger'
+    prefix = '+'
+  }
+
+  return (
+    <Badge variant={variant}>
+      {prefix}
+      {value.toFixed(2)}%
```

Comment:
just checking: we're fine with +1.00 or 0.00?

Reply:
still need to double check with Product, great callout though -- will add a ticket for polish to the epic

Reply:
Created https://github.com/codecov/engineering-team/issues/2662